tests/fixtures/app/typescript-embroider/package.json

```diff
@@ -38,36 +48,12 @@
     "@glint/environment-ember-loose": "^1.4.0",
     "@glint/template": "^1.4.0",
     "@tsconfig/ember": "^3.0.8",
-    "@types/ember": "^4.0.11",
-    "@types/ember-data": "^4.4.16",
-    "@types/ember-data__adapter": "^4.0.6",
-    "@types/ember-data__model": "^4.0.5",
-    "@types/ember-data__serializer": "^4.0.6",
-    "@types/ember-data__store": "^4.0.7",
-    "@types/ember__application": "^4.0.11",
-    "@types/ember__array": "^4.0.10",
-    "@types/ember__component": "^4.0.22",
-    "@types/ember__controller": "^4.0.12",
-    "@types/ember__debug": "^4.0.8",
-    "@types/ember__destroyable": "^4.0.5",
-    "@types/ember__engine": "^4.0.11",
-    "@types/ember__error": "^4.0.6",
-    "@types/ember__helper": "^4.0.8",
-    "@types/ember__modifier": "^4.0.9",
-    "@types/ember__object": "^4.0.12",
-    "@types/ember__owner": "^4.0.9",
-    "@types/ember__polyfills": "^4.0.6",
-    "@types/ember__routing": "^4.0.22",
-    "@types/ember__runloop": "^4.0.10",
-    "@types/ember__service": "^4.0.9",
-    "@types/ember__string": "^3.16.3",
-    "@types/ember__template": "^4.0.7",
-    "@types/ember__test": "^4.0.6",
-    "@types/ember__utils": "^4.0.7",
     "@types/qunit": "^2.19.10",
     "@types/rsvp": "^4.0.9",
     "@typescript-eslint/eslint-plugin": "^7.18.0",
     "@typescript-eslint/parser": "^7.18.0",
+    "@warp-drive-types/core-types": "0.0.0-alpha.94",
+    "@warp-drive/core-types": "0.0.0-alpha.94",
```

Comment:
I think we only want 1 of these, depending if we're going for real packages or type packages.